package.json

```diff
@@ -62,6 +62,7 @@
     "eslint-plugin-promise": "^6.1.1",
     "jest": "^29.7.0",
     "prettier": "^3.2.4",
+    "ts-node": "^10.9.2",
```

Comment:
I don't understand why we use this diff. Shouldn't `ts-node` be globally installed?

Reply:
It's there in the app repo also. So ts-node should be installed as part of dev setup?

Reply:
Disregard this comment; I think you do need `ts-node` in the dev dependencies.